ament_clang_tidy/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ament_clang_tidy</name>
+  <version>6.0.0</version>
```

Comment:
I'd reduce this version number to 1.0.0, since this is new.  Ditto for setup.py.

Reply:
The version number must match the version of the other packages in this repo.

Reply:
Oops, yeah, forgot about this.  Never mind this comment then :).